.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,38 @@
 
+---
+**PR checklist**
+
+The following checklist shall help the PR's author, the reviewers and maintainers to ensure the quality of this project.
```

Comment:
Simplify: shall help -> helps.

Reply:
I used the "shall help" by purpose, because I don't know if the list helps, I don't want to create a feeling of pressure. That's also why I listed author, reviewers and maintainers - so that an (new or inexperienced) contributor hopefully feels not constrained